src/tls/test-cockpit-certificate-ensure.c

```diff
@@ -314,7 +314,7 @@ static const TestCase case_bad_file = {
 static const TestCase case_bad_file2 = {
   .files = bad_files2,
 
-  .check_stdout = "",
+  .check_stdout = "Unable to read*",
```

Comment:
This is a symptom of the above "should go to stderr" comment. The stderr check already covers having a more precise error message.